### ISSUE TITLE
git tag for documentation(`docs_2.7.0`) was detected by carthage as latest version.

### ISSUE THREAD

**Issue by r-plus** [aws-amplify/aws-sdk-ios#1101]
Hi.

`docs_2.7.0` tag was detected by carthage as latest version since `docs_X.X.X` git tag was published.

```bash
$ cat Cartfile
github "aws/aws-sdk-ios" ~> 2.4

$ carthage outdated
aws-sdk-ios "2.6.32" -> "docs_2.7.0" (Latest: "docs_2.7.0")
```

`docs_X.X.X` tag is not include source code, so its build will failed.
Carthage [skip leading any characters](https://github.com/Carthage/Carthage/blob/0.31.2/Source/CarthageKit/Version.swift#L64) before `0-9.`, so `docs_2.7.0` was interpreted as 2.7.0 semantic version.

I suggest `docs` as suffix(e.g. `2.7.0_docs`) or any other non-semantic version tag to be ignored by carthage.(not tested)

**Comment by rohandubal** [comment]
Thanks for reporting this @r-plus 

As an immediate step, I have removed the documentation tags so that Carthage does not fail. I will tag the documentation with the correct format once I test it with Carthage.

I will update the thread once we have tagged the documentation with the right tag.

Thanks,
Rohan

**Comment by rohandubal** [comment]
Hello @r-plus 

We have now resolved this by using a tag format [`2.8.1_api_docs`](https://github.com/aws-amplify/aws-sdk-ios/tree/2.8.1_api_docs) (similar to what you suggested) and it is not impacting Carthage.

Thanks,
Rohan
